lib/views/pages/members/members.dart

```diff
@@ -22,13 +24,15 @@ class Organizations extends StatefulWidget {
 }
 
 class _OrganizationsState extends State<Organizations> {
+
+  String currentOrgID;
   List alphaMembersList = [];
   int isSelected = 0;
   Preferences preferences = Preferences();
 
 
   //providing initial states to the variables
-  initState() {
+   initState( )  {
```

Comment:
Sorry, I did not understand requirement for this change ?

Reply:
no sir there is no need of this it is just by mistake i clicked on spacebar to look spatial